### DISCUSSION TITLE
Fail2ban

### DISCUSSION THREAD

**Discussion by joelang1699** [photoview/photoview#628] [Ideas]
Any chance of being able to log failed login attempts for fail2ban to read? Very useful for public facing.

**Comment by githlp** [comment]
my fail2ban setup:

add following to /etc/fail2ban/jail.local
```
[photoview]
enabled = true
filter = photoview
logpath = /your-path-to/photoview.access.log

```

then create this configuration file:
 /etc/fail2ban/filter.d/photoview.conf
```
[Definition]
failregex = ^<HOST> -.*POST /api/graphql HTTP/2.0" 200 123
ignoreregex =
```
